bika/lims/browser/worksheet/views/results.py

```diff
@@ -2,51 +2,52 @@
 
 # This file is part of Bika LIMS
 #
-# Copyright 2011-2016 by it's authors.
+# Copyright 2011-2017 by it's authors.
 # Some rights reserved. See LICENSE.txt, AUTHORS.txt.
 
+from bika.lims import bikaMessageFactory as _
+from bika.lims.browser import BrowserView
+from bika.lims.browser.worksheet.tools import (checkUserAccess,
+                                               showRejectionMessage)
+from bika.lims.browser.worksheet.views import (AnalysesTransposedView,
+                                               AnalysesView)
+from bika.lims.config import WORKSHEET_LAYOUT_OPTIONS
+from bika.lims.utils import getUsers, tmpID
 from plone.app.layout.globals.interfaces import IViewView
 from Products.Archetypes.config import REFERENCE_CATALOG
 from Products.Archetypes.public import DisplayList
-from Products.CMFPlone.i18nl10n import ulocalized_time
-from Products.CMFPlone.utils import _createObjectByType
-from Products.CMFPlone.utils import safe_unicode
 from Products.CMFCore.utils import getToolByName
+from Products.CMFPlone.utils import _createObjectByType, safe_unicode
 from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
 from zope.interface import implements
 
-from bika.lims import bikaMessageFactory as _
-from bika.lims.config import WORKSHEET_LAYOUT_OPTIONS
-from bika.lims.utils import t
-from bika.lims.browser import BrowserView
-from bika.lims.browser.worksheet.tools import checkUserAccess
-from bika.lims.browser.worksheet.tools import showRejectionMessage
-from bika.lims.browser.worksheet.views import AnalysesTransposedView
-from bika.lims.browser.worksheet.views import AnalysesView
-from bika.lims.utils import getUsers, tmpID
-
 
 class ManageResultsView(BrowserView):
     implements(IViewView)
     template = ViewPageTemplateFile("../templates/results.pt")
+
     def __init__(self, context, request):
         BrowserView.__init__(self, context, request)
         self.getAnalysts = getUsers(context, ['Manager', 'LabManager', 'Analyst'])
         self.layout_displaylist = WORKSHEET_LAYOUT_OPTIONS
 
     def __call__(self):
+
         # Deny access to foreign analysts
-        if checkUserAccess(self.context, self.request) == False:
+        if checkUserAccess(self.context, self.request) is False:
             return []
 
         showRejectionMessage(self.context)
 
         self.icon = self.portal_url + "/++resource++bika.lims.images/worksheet_big.png"
 
-        # Worksheet Attachmemts
-        # the expandable form is handled here.
+        # TODO: Move this form handler into bika.lims.browser.attachment and
+        #       change the form action of
+        #       bika.lims.browser.viewlets.templates.worksheet_attachments.pt
+        #       accordingly!
+        #       Relevant issue: https://github.com/senaite/bika.lims/issues/521
```

Comment:
I suggest to create an issue (also if labeled as `low-priority`) too. Could be labeled with `architecture`